docs/tutorial/acquiring_data/jsoc.rst

```diff
@@ -6,9 +8,10 @@ Joint Science Operations Center (JSOC) contains data products from the Solar Dyn
 These data are available from the JSOC database, which can be directly accessed by the online `JSOC interface <http://jsoc.stanford.edu/ajax/lookdata.html>`__.
 
 sunpy's JSOC Client provides an easier interface to query for JSOC data and make export requests.
-It uses `drms module <https://docs.sunpy.org/projects/drms>`_ as its backend, and exposes a similar API as the VSO Client.
+It uses `drms` as its backend.
```

Comment:
Changing this to `drms` probably changes this from linking to the front page of the docs to linking to the API docs for the top level of the package. Is that actually what we want?

Reply:
Good point.